spec/models/organization_spec.rb

```diff
@@ -21,8 +23,39 @@
     end
   end
 
+  context 'scopes' do
+    before(:each) do
+      @organization1 = create(:organization, name: 'Spritle')
+      @organization2 = create(:organization, name: 'Foo')
+      @project1 = create(:project, organization_id: @organization1.id)
+      @project2 = create(:project, organization_id: @organization1.id, is_active: false)
+      @project3 = create(:project, organization_id: @organization1.id, is_approved: true)
+      @job = create(:job, organization_id: @organization1.id)
+    end
+
+    describe 'approved scope' do
+      it 'should return all organization which has approved projects' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.